### PR TITLE
Syntax Prettify

### PR DESCRIPTION
**Pull request by ctan-g**
This PR prettifies Python code.
- Use `yapf` to auto prettify, using Google style, with the exception of setting column limit to 119 to match OT.
- Added other manual fixes, such as breaking longer lines, modifying constant/function names, and unifying single/double quotes.
- Improved `pylint` score from less than 0 to about 9.5 using Google's lint rc file (Fixed more than 20k warnings, still has about 3k to fix).
- Added lint step, but set it to pass in spite of error for the time being.
- Ran through all 118 Silk test cases to verify that no functionality has been broken.

### CODE REVIEW

**Comment by priyankaCh22** [comment]
Even after adding fix PR 55 during test run getting errors (tried 1 test only):
"===================== HARDWARE TEST SET-UP ERRORS ======================
========================================================================"
setUpClass (openthread.ot_test_form_network.TestFormNetwork)
Traceback (most recent call last):
  File "/home/pi/silk/env/lib/python3.6/site-packages/silk-1.0.0-py3.6.egg/silk/tests/testcase.py", line 260, in wrapper
    func(*args, **kwargs)
  File "/home/pi/silk/silk/tests/openthread/ot_test_form_network.py", line 52, in setUpClass
    cls.hardwareSelect()
  File "/home/pi/silk/silk/tests/openthread/ot_test_form_network.py", line 34, in hardwareSelect
    cls.router = ffdb.ThreadDevBoard()
  File "/home/pi/silk/env/lib/python3.6/site-packages/silk-1.0.0-py3.6.egg/silk/node/fifteen_four_dev_board.py", line 166, in __init__
    NetnsController.__init__(self)
  File "/home/pi/silk/env/lib/python3.6/site-packages/silk-1.0.0-py3.6.egg/silk/device/netns_base.py", line 70, in __init__
    self.create_netns()
  File "/home/pi/silk/env/lib/python3.6/site-packages/silk-1.0.0-py3.6.egg/silk/device/netns_base.py", line 82, in create_netns
    self.netns = os.path.basename(self.device_path)
  File "/usr/lib/python3.6/posixpath.py", line 146, in basename
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType

**Comment by ctan-g** [comment]
@priyankaCh22 Looks like the error is not related to this PR, but to #51 itself. I'll make another change to #55.

Edit: added. Should add more testing to catch things like this :crystal_ball:

File: silk/config/wpan_constants.py [hunk 1]
```diff
@@ -14,180 +14,172 @@
...
-STATE_COMMISSIONED                             =  '"offline:commissioned"'
-STATE_ASSOCIATING                              =  '"associating"'
-STATE_CREDENTIALS_NEEDED                       =  '"associating:credentials-needed"'
-STATE_ASSOCIATED                               =  '"associated"'
-STATE_ISOLATED                                 =  '"associated:no-parent"'
-STATE_NETWAKE_ASLEEP                           =  '"associated:netwake-asleep"'
-STATE_NETWAKE_WAKING                           =  '"associated:netwake-waking"'
-
-#-----------------------------------------------------------------------------------------------------------------------
+STATE_UNINITIALIZED = "uninitialized"
```
Comment (qilinkang) [inline silk/config/wpan_constants.py]: Removing '' will cause the verification failed.

File: silk/node/fifteen_four_dev_board.py [hunk 2]
```diff
@@ -529,51 +523,54 @@ def do_flash_efr32(delegates):
...
-           `prefix_len` is an `int` specifying the prefix length.
-           NOTE: this method uses linux `ip` command.
+        `address` should be string containing the IPv6 address.
+        `prefix_len` is an `int` specifying the prefix length.
+        NOTE: this method uses linux `ip` command.
         """
-
-        cmd = 'ip -6 address add ' + address + '/{} dev '.format(prefix_len) + self.thread_interface
+        interface = self.thread_interface
+        cmd = f"ip -6 address add {address} /{prefix_len} dev {interface}"
```
Comment (qilinkang) [inline silk/node/fifteen_four_dev_board.py]: Need to remove the space: {address} /{prefix_len} -> {address}/{prefix_len}

File: silk/node/fifteen_four_dev_board.py [hunk 3]
```diff
@@ -529,51 +523,54 @@ def do_flash_efr32(delegates):
...
-           `prefix_len` is an `int` specifying the prefix length.
-           NOTE: this method uses linux `ip` command.
+        `address` should be string containing the IPv6 address.
+        `prefix_len` is an `int` specifying the prefix length.
+        NOTE: this method uses linux `ip` command.
         """
-
-        cmd = 'ip  -6 address del ' + address + '/{} dev '.format(prefix_len) + self.thread_interface
+        interface = self.thread_interface
+        cmd = f"ip -6 address del {address} /{prefix_len} dev {interface}"
```
Comment (qilinkang) [inline silk/node/fifteen_four_dev_board.py]: Need to remove the space: {address} /{prefix_len} -> {address}/{prefix_len}

**Review by priyankaCh22 — APPROVED** [review]
LGTM

**Review by qilinkang — APPROVED** [review]
All tests are passed.

**Review by jwhui — APPROVED** [review]

**Pull request merged**